src/test/java/TestLargestElement.java

```diff
@@ -0,0 +1,15 @@
+import Arrays.LargestElement;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class TestLargestElement {
+    @Test
+    public void testLargestElement()
+    {
+        int arr []={5,3,6,1,7,9};
+        int expected=9;
+        int actual=LargestElement.largestElement(arr);
+        assertEquals(expected,actual);
+    }
+}
```

Comment:
can you write few other test methods. I see you are just doing test with happy paths only.

int[] arr = {2,2,2,2};
int[] arr = {}

Try to write test methods for above inputs